querydsl-spatial2/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.querydsl</groupId>
+        <artifactId>querydsl-root</artifactId>
+        <version>4.4.1-SNAPSHOT</version>
```

Comment:
Now the PR is up to date, this should be updated  to `5.0.0-SNAPSHOT`. Then the build will work again.